v3/plugins/cdr/code-server/3.6.2/meta.yaml

```diff
@@ -5,11 +5,11 @@ version: 3.6.2
 type: Che Editor
 displayName: code-server
 title: Visual Studio Code - Web (code-server)
-description: An open source distribution of Visual Studio Code as a cloud IDE
+description: A open source distribution of Visual Studio Code as a cloud IDE
```

Comment:
you undid a typo correction?

Reply:
so please next time, do separate commits when you do something not related to the title of the PR
because here it's highlighting that you did changes in a commit that was not related to update to quay.io